app/mailers/user_notifier.rb

```diff
@@ -0,0 +1,9 @@
+class UserNotifier < ApplicationMailer
+  default :from => 'hipsterbites@gmail.com'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.